addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -285,6 +285,10 @@ export class MassMailingHtmlField extends HtmlField {
             this._themeParams = Array.from(displayableThemes).map((theme) => {
                 const $theme = $(theme);
                 const name = $theme.data("name");
+                // TODO remove in master and remove apply the update in xml directly
```

Comment:
"and remove apply"? 🤔

Reply:
I knew about it and I thought I fixed it already, fixed now